src/uu/join/src/join.rs

```diff
@@ -444,9 +445,11 @@ impl<'a> State<'a> {
 
         if diff == Ordering::Greater {
             eprintln!(
-                "{}:{}: is not sorted",
+                "{}: {}:{}: is not sorted: {}",
+                uucore::util_name(),
                 self.file_name.maybe_quote(),
-                self.line_num
+                self.line_num,
+                String::from_utf8_lossy(&line.string)
```

Comment:
This change (while correct) broke the `wrong_line_order` test in `tests/by_util/test_join.rs`. Could you adapt the test so that it passes again?

Reply:
Ah! Thanks for pointing this out. I was only running `cargo test` in the `src/uu/join/` directory so I missed this one.